src/components/NcListItem/NcListItem.vue

```diff
@@ -833,8 +836,8 @@ export default {
 			min-width: 0;
 		}
 		.list-item-content__details {
-			display: flex;
```

Comment:
Question: don't we need this in order to still use `flex-direction` here?

Reply:
See line 824 - `display: flex` is already applied